site/content/en/references/kustomize/kustomization/buildmetadata/_index.md

```diff
@@ -0,0 +1,308 @@
+---
+title: "buildMetadata"
+linkTitle: "buildMetadata"
+type: docs
+weight: 18
+description: >
+    Specify options for including information about the build in annotations or labels. 
+---
+
+The `buildMetadata` field is a list of strings. The strings can be one of three builtin
+options that add some metadata to each resource about how the resource was built. 
+
+These options are:
+
+- `managedByLabel`
+- `originAnnotations`
+- `transformerAnnotations`
+
+It is possible to set one or all of these options in the kustomization file:
+
+It is possible for to set both `transformerAnnotations` and `originAnnotations`:
```

Comment:
Remove? Same as above but missing managedByLabel.

Reply:
Ah my bad, copy paste mistake. Thanks for the catch!